.gitignore

```diff
@@ -14,7 +14,6 @@ platform/android/java/local.properties
 platform/android/java/project.properties
 platform/android/java/libs/*
 platform/android/java/assets
-platform/android/java/.idea/*
```

Comment:
Moved to top level, similar to other IDEs, as Jetbrains has products focused on C/C++, Python, and C#